src/spikeinterface/core/baserecording.py

```diff
@@ -746,6 +746,30 @@ def _select_segments(self, segment_indices):
 
         return SelectSegmentRecording(self, segment_indices=segment_indices)
 
+    def get_channel_locations(
+        self,
+        channel_ids: list | np.ndarray | tuple | None = None,
+        axes: "xy" | "yz" | "xz" = "xy",
```

Comment:
```suggestion
        axes: "xy" | "yz" | "xz" | "xyz" = "xy",
```
based on the possible return we are missing the 3d option. Or the return needs to be fixed!